pathways/simulation.py

```diff
@@ -272,7 +327,10 @@ def run_simulation(
             output_f280_file=output_f280_file,
             verbose=verbose,
             pretty=pretty,
+            detailed=detailed,
         )
+        if detailed and i == 0:
+            details = result.details
```

Comment:
You should document in the docstring that details are for the first simulation only while totals are for all.